AdobeStockClientApi/Api/ClientInterface.php

```diff
@@ -56,4 +56,19 @@ public function getFullEntitlementQuota(): UserQuotaInterface;
      * @return bool
      */
     public function testConnection(string $apiKey = null): bool;
+
+    /**
+     * Invokes licensing image operation via Adobe Stock API
+     *
+     * @param int $contentId
+     */
+    public function licenseImage(int $contentId);
```

Comment:
return type is missing. It's 'License' in the implementation, however, I think it should be void